Assets/Scripts/Managers/HuntingInputManager.cs

```diff
@@ -94,7 +94,7 @@ private void CloseWeaponWheel()
 		weaponWheelController.CloseWeaponWheel();
 	}
 
-	public bool WeaponWheelisOpen()
+	public bool WeaponWheelIsOpen()
 	{
 		return weaponWheelController.getOpen();
 	}
```

Comment:
Compile issues:
`WeaponWheelIsOpen` is treated like a property elsewhere
`GetOpen` capitalization not fixed
```suggestion
	public bool WeaponWheelIsOpen => weaponWheelController.GetOpen();
```